.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 <h3>New features since last release</h3>
 
+* Added functionality to compute the sparse matrix representation of a `qml.Hamiltonian` object.
```

Comment:
It's always good to add a short code snippet with an example of how to use this new functionality